pages/signup.html

```diff
@@ -0,0 +1,107 @@
+        </div>
+        <nav class="header-nav">
+            <ul class="header-list">
+                <a href="./login.html" class="header-item">
+                    <button class="btn btn-primary">Login</button>
+                </a>
+                <a href="./wishlist.html" class="header-item">
+                    <p class="badge position-relative">
+                        <i class="fa fa-solid fa-heart badge-icon"></i>
+                        <span class="badge-content position-absolute sm-badge">12</span>
+                    </p>
+                    <span class="header-item-text">Wishlist</span>
+                </a>
+                <a href="./" class="header-item">
+                    <p class="badge position-relative">
+                        <i class="fas fa-shopping-cart badge-icon"></i>
+                        <span class="badge-content position-absolute sm-badge">1</span>
+                    </p>
+                    <span class="header-item-text">Cart</span>
+                </a>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <div class="auth-wrapper rounded-5">
+            <div class="auth-body">
+                <div class="auth-head">
+                    Sign-Up
+                </div>
+                <div class="form-container">
+                    <div class="input-container">
+                        <label for="name" class="input-label">Full Name <span class="required-mark">*</span></label>
+                        <input type="text" id="name" class="input-field" placeholder="XYZ" required>
+                    </div>
+                    <div class="input-container">
+                        <label for="email" class="input-label">Email Address <span
+                                class="required-mark">*</span></label>
+                        <input type="email" id="email" class="input-field" placeholder="xyz@example.com" required>
+                    </div>
+                    <div class="input-container">
+                        <label for="password" class="input-label">Password <span class="required-mark">*</span></label>
+                        <input type="password" for="password" class="input-field" required>
+                    </div>
+                    <div class="input-container">
+                        <label for="conf-password" class="input-label">Confirm Password <span
+                                class="required-mark">*</span></label>
+                        <input type="password" for="conf-password" class="input-field" required>
+                    </div>
+                    <div class="auth-form-subsection">
+                        <div class="input-selector">
+                            <input type="checkbox" name="filter" id="" class="input-selector-field" id="checkbox-1">
+                            <label for="checkbox-1" class="input-selector-label">I accept all Terms & Conditions <span

```

Comment:
We can put a label before input.

Reply:
Yeah, we can but I want to have the label after the checkbox.